modules/pytket-braket/pytket/extensions/braket/backends/braket.py

```diff
@@ -219,6 +220,7 @@ def __init__(
             default: "quantum-simulator"
         :param provider: provider name from device ARN (e.g. "ionq", "rigetti", ...),
             default: "amazon"
+        :param aws_session: braket AwsSession object, to pass credentials in
```

Comment:
Can we add something like "if not configured on local machine", so as not to give the impression people need to use this?